### DISCUSSION TITLE
Running Factor IDE on LiveCD Puppy 14.04, 16.04 Xenial, Bionic (32)

### DISCUSSION THREAD

**Discussion by forthuser** [factor/factor#2460] [Q&A]
Hi,
My resultat after assemble Factor
https://i.postimg.cc/3x6Z5sKP/Factor.png

Why, before running need starts?
libgtkglext1_1.2.0-3.2fakesync1ubuntu1_i386.deb
ibgtkglext1-dev_1.2.0-3.2fakesync1ubuntu1_i386.deb

Libudis86 working on Ubuntu 14.04, 16.04 on Factor? 
https://github.com/vmt/udis86
On site Factor not finding Libudis86.so or deb package

P.S. My factor execution file http://sendfile.su/1604411

**Comment by mrjbq7** [comment]
What is your question?

**Comment by forthuser** [comment]
Will the official Factor working on Puppy Linux ?

P.S. I can't collect libudis86.so, if where is this file ready for Linux 32? 
(and other libopenal.so.0, libalut.so, libraylib.so) 

Not Working on Puppy Linux applications:  Hacker News, CPU 8080 Emulator, Wikipedia
on https://planet.factorcode.org/

**Comment by mrjbq7** [comment]
It should, is it not working?

http://udis86.sourceforge.net/

**Reply by forthuser** [reply]
I do not know, I did not assemble the .so automatic, except .a
Libtool collected, but the test could not run:
`: add5 ( x -- y ) 5 + ;

\ add5 disassemble` 

P.S. Puppy Linux have GCC on devxx.sfc

**Comment by mrjbq7** [comment]
So factor runs fine just udis86 isn’t working?

We load the udis86.so Library, so it needs to be available via dlopen.

**Reply by forthuser** [reply]
Perhaps I do not have libudis86.so (32) 

P.S. Factor working fine!
On Screen Image 0.99 x86,32 (2100) Apr 20 2021 heads/master-81b4cdfcc1
GCC 5.4.0 20160809

**Comment by mrjbq7** [comment]
Out of curiosity, did you download the nightly build binary or build it yourself?

**Reply by forthuser** [reply]
Assembled from nightly sources on LiveCD Linux XenialPup (on RAM Storage Puppy) 

P.S. The file (Factor) is applied in the first message.

**Reply by forthuser** [reply]
https://github.com/factor/factor/discussions/2460

**Reply by alex-ilin** [reply]
It's so nice to see my tip of the day on the screenshot there : ))
